myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java

```diff
@@ -212,6 +212,9 @@ public String getFrameworkName() {
   }
 
   public String getFrameworkRole() {
+    if (frameworkRole.contains("/")) {
```

Comment:
Hi @ hokiegeek2, sorry but I think this does not solve the problem. Maybe it's because the code has changed and the patch test does not make sense with the current code. The current problem is that we are treating the default role 'asterisk' as if it were a real role and the behavior has to be as if it were not a role. When the role is 'asterisk', "frameworkRole.contains (" / ")" returns false and never executes the statement "return frameworkRole.replace (" / "," ");", then it continues to fail. I think that the error is produced by the treatment of the role 'asterisc' in "ResourceOfferContainer.setScalarValues", it would be necessary that the behavior of this function with role 'asterisk' was equal to a roleless one. What do you think?